Add Proper License to com.jme3.anim files

ItsMike54 opened the pull request:
In order to chip away at this issue, https://github.com/jMonkeyEngine/jmonkeyengine/issues/1001
I've added the proper jME license to the Anim files written by @Nehon (or at least the ones that he committed to, excluding the ones by pspeed which already have the new licenses) . 
Now in the previous discussion on that issue, it seemed as if @stephengold was attempting to contact nehon to get his permission. If Nehon replies, then we'll follow from there, but if he doesn't, I say that since he contributed as an engine leader, it would be fine to go ahead and add the licenses.

stephengold commented on jme3-core/src/main/java/com/jme3/anim/SkinningControl.java:
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2009-2019 jMonkeyEngine
```
This change seems unnecessary.

stephengold commented on jme3-core/src/main/java/com/jme3/anim/AnimTrack.java:
```diff
@@ -1,3 +1,34 @@
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Nehon didn't claim authorship of this file.

ItsMike54 replied:
Ohh....well should that be removed then? Or try to find the author?

stephengold commented on jme3-core/src/main/java/com/jme3/anim/ArmatureMask.java:
```diff
@@ -1,3 +1,34 @@
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Nehon didn't claim authorship of this file.

stephengold commented on jme3-core/src/main/java/com/jme3/anim/MatrixJointModelTransform.java:
```diff
@@ -1,3 +1,34 @@
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Nehon didn't claim authorship of this file.

stephengold commented on jme3-core/src/main/java/com/jme3/anim/SeparateJointModelTransform.java:
```diff
@@ -1,3 +1,34 @@
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Nehon didn't claim authorship of this file.

ItsMike54 commented:
@stephengold , simply remove those headers then? Or try to find their author?

stephengold commented on jme3-core/src/main/java/com/jme3/anim/Weights.java:
```diff
@@ -1,3 +1,34 @@
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Nehon didn't claim authorship of this file.

stephengold commented on jme3-core/src/main/java/com/jme3/anim/tween/ContainsTweens.java:
```diff
@@ -1,3 +1,34 @@
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Nehon didn't claim authorship of this file.

stephengold commented on jme3-core/src/main/java/com/jme3/anim/tween/action/Action.java:
```diff
@@ -1,3 +1,34 @@
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Nehon didn't claim authorship of this file.

stephengold commented on jme3-core/src/main/java/com/jme3/anim/tween/action/BaseAction.java:
```diff
@@ -1,3 +1,34 @@
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Nehon didn't claim authorship of this file.

stephengold commented:
> @stephengold , simply remove those headers then? Or try to find their author?

I don't know what to do with these files. Their style matches Nehon's, so I'm 90% sure he wrote them. But he's the only person who knows for sure, and as I said, he hasn't responded to my inquiries. Perhaps someone else will have better luck contacting him.

ItsMike54 commented:
Well assumptions bite me in the ass, and I'll remove those right now.

stephengold commented on jme3-core/src/main/java/com/jme3/anim/AnimTrack.java:
```diff
@@ -8,5 +8,4 @@
     public void getDataAtTime(double time, T store);
     public double getLength();
 
-
```
I don't see any reason to modify AnimTrack.java

stephengold commented on jme3-core/src/main/java/com/jme3/anim/tween/action/BlendAction.java:
```diff
@@ -1,3 +1,34 @@
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Nehon didn't claim authorship.

stephengold commented on jme3-core/src/main/java/com/jme3/anim/tween/action/BlendSpace.java:
```diff
@@ -1,3 +1,34 @@
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Authorship uncertain.

stephengold commented on jme3-core/src/main/java/com/jme3/anim/tween/action/BlendableAction.java:
```diff
@@ -1,3 +1,34 @@
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
```
Authorship uncertain.

ItsMike54 commented:
@stephengold , I went ahead and reviewed the files, removing all that Nehon didn't expressly state his authorship in. I did add licenses to some that I missed previously that he did state authorship of.

stephengold commented:
Please take another look at LinearBlendSpace.java

ItsMike54 commented:
Obviously I can't seem to make a perfect PR just yet, but LinearBlendSpace.java is fixed.

stephengold commented:
As far as I can tell, this PR just needs the one-line change to AnimTrack.java and then it will be ready to merge.

@ItsMike54 have you abandoned this PR?

ItsMike54 commented:
@stephengold Over the past week I've been trying to make progress on a project and I don't have much time to do so, so I abandoned it for that time.
But since it was brought up again, I reverted the AnimTrack.java interface back to its original state before I modified it.

stephengold commented:
It's not back to its original state.

ItsMike54 commented:
@stephengold , I deleted the AnimTrack.java from my branch and uploaded the AnimTrack.java interface that is currently on Master jME branch. That file is completely unmodified, and I don't know what else to say about it.
